dsd/constraints.py

```diff
@@ -617,18 +617,18 @@ def satisfies_sequence_constraints(self, sequence: str) -> bool:
         """
         return all(constraint(sequence) for constraint in self.sequence_constraints)
 
-    def find_sequence_neighbors(self, steps: int, previous_sequence: str) -> List[str]:
+    def find_steps_distance_sequences(self, steps: int, previous_sequence: str) -> List[str]:
         """
-        Makes an adjacency list of sequences some Hamming distance away from the previous sequence.
+        Makes a list of sequences some Hamming distance away from the previous sequence.
         The sequence that replaces the previous sequence will be randomly chosen from this list.
         """
-        adjacency_list= []
+        neighbors_list= []
```

Comment:
I'd just call it `neighbors` instead of `neighbors_list`. Usually the only time I put the name of a collection in the variable name is to differentiate two of them, e.g. `neighbors_list` versus `neighbors_set` or `neighbors_dict`, if you had both. But just `neighbors` is a more standard variable name than `neighbors_list`.